### PR TITLE
Refactor/dp3129 rabbitmq timeout

### PR DESCRIPTION
**Pull request by youwithouto**

### CODE REVIEW

File: notification/notification.go [hunk 1]
```diff
@@ -135,12 +140,14 @@ func (manager *AppMobileNotificationManager) SendDataNotification(ctx context.Co
...
-	return manager.sendDataNotification(ctx, tokens, title, message, customData)
+	return manager.sendDataNotifications(ctx, tokens, title, message, customData)
 }
 
+// BroadcastDataNotification sends a push notification to all available tokens
+// On error, it returns a pointer to DataNotificationError
 func (manager *AppMobileNotificationManager) BroadcastDataNotification(ctx context.Context, title, message string, customData map[string]interface{}) (err error) {
 
-	tokens, err := manager.findTokens(nil)
+	tokens, err := manager.FindTokens(nil)
```
Comment (jakequade-hsm) [inline notification/notification.go:150]: nit: why is this snake-cased vs the others?

**Review by jakequade-hsm — APPROVED** [review]

File: notification/notification.go (hunk 1 above)
Comment (youwithouto) [reply notification/notification.go:150]: > nit: why is this snake-cased vs the other functions?

Where is the snake-case?

Comment (jakequade-hsm) [reply notification/notification.go:150]: manager.FindTokens

Comment (youwithouto) [reply notification/notification.go:150]: > `manager.FindTokens` -> `F`

Using the uppercase 'F' makes that method public and callable by code outside of its containing package, and in this case, it is called by the code in the HCF service when broadcasting notifications.

**Pull request merged**